models/MediaTreeGenerator.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
+ *
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoDevTools\models;
+
+use common_ext_ExtensionsManager;
+use core_kernel_classes_Class as RdfClass;
+use oat\taoMediaManager\model\MediaService;
+
+class MediaTreeGenerator extends AbstractTreeGenerator
+{
+
+    protected function provideOptions(): array
+    {
+        return
+            array_merge(
+                parent::provideOptions(),
+                [
+                    self::OPTION_ROOT_CLASS => [
+                        'longPrefix' => 'root-class',
+                        'prefix' => 'k',
+                        'required' => false,
+                        'description' => 'Root class',
+                        'defaultValue' => MediaService::ROOT_CLASS_URI
+                    ],
+                ]
+            );
+    }
+
+
+    protected function provideDescription(): string
+    {
+        return 'Tool to generate a tree of media assets';
+    }
+
+    protected function generateItem(RdfClass $class, int $count): void
+    {
+        for ($i = 0; $i < $count; $i++) {
+            $sampleFile = $this->getAssetFilePath();
+
+            $mediaResourceUri = $this->getAssetImportService()->createMediaInstance(
+                $sampleFile,
+                $class->getUri(),
+                'en-US',
```

Comment:
We should use custom platform config instead of hardcoded language (I would provide by DI)